app-admin/logstash-bin/files/logstash-plugin.eselect

```diff
@@ -0,0 +1,80 @@
+# -*-eselect-*-  vim: ft=eselect
+# Copyright 2005-2017 Gentoo Foundation
+# Distributed under the terms of the GNU GPL version 2 or later
+
+DESCRIPTION="Manage user installed plugins of logstash"
+MAINTAINER="hydrapolic@gmail.com"
+
+PLUGINS="/var/lib/logstash/plugins-list.txt"
+LS_INSTALL_DIR="/opt/logstash"
+
+ls_plugin_wrapper() {
+  cd "${LS_INSTALL_DIR}" || die
+  JARS_SKIP=true ./bin/logstash-plugin "$1" "$2"
+}
+
+### install action
+describe_install() {
+  echo "Install plugin"
+}
+
+describe_install_parameters() {
+  echo "<plugin>"
+}
+
+do_install() {
+  local x
+  for x; do
+    echo "$1" >> ${PLUGINS}
+    ls_plugin_wrapper install "$1"
+    shift
```

Comment:
`for x` already iterates through arguments. I don't think `shift` interferes with that but if it did, you would skip every second argument.